### ISSUE TITLE
polyfill.Fetch object request body (application/json ) being converted to string

### ISSUE THREAD

**Issue by mikemonteith** [wkh237/react-native-fetch-blob#160]
It seems that fetch() calls with a JSON body, the JSON is being converted to a string.

This is the whatwg-fetch way of doing a JSON post:

```
fetch(URL, {
  method: 'POST',
  headers: {
    'Content-Type': 'application/json',
  },
  body: JSON.stringify({abc: 123}),
});
```

I have also tried

```
fetch(URL, {
  method: 'POST',
  headers: {
    'Content-Type': 'application/json',
  },
  body: {abc: 123},
});
```

I think that polyfill.Fetch is converting my JSON-string into a string-containing-a-JSON-string.

**Comment by mikemonteith** [comment]
https://github.com/wkh237/react-native-fetch-blob/blob/master/src/polyfill/Fetch.js#L50

Should this line be `else if (typeof body === 'object')`?

**Comment by wkh237** [comment]
@mikemonteith , FYI, I've published `0.10.0-beta.4` which includes your PR 👍
